NDB.Covid19/NDB.Covid19/Locales/nn.json

```diff
@@ -252,20 +270,24 @@
   "NOTIFICATION_BACKGROUND_FETCH_DESCRIPTION": "Smittestopp undersøker om du har vore i nærleiken av brukarar som har varsla om at dei er smitta.",
   "NOTIFICATION_BLUETOOTH_OFF_TITLE": "Smittestopp er inaktiv",
   "NOTIFICATION_BLUETOOTH_OFF_DESCRIPTION": "Opne appen og reaktiver Smittstopp",
+  "NOTIFICATION_LOCATION_OFF_TITLE": "Smittestopp er inaktiv",
+  "NOTIFICATION_LOCATION_OFF_DESCRIPTION": "Opne appen og reaktiver Smittstopp",
+  "NOTIFICATION_BLUETOOTH_AND_LOCATION_OFF_TITLE": "Smittestopp er inaktiv",
+  "NOTIFICATION_BLUETOOTH_AND_LOCATION_OFF_DESCRIPTION": "Opne appen og reaktiver Smittstopp",
   "SETTINGS_GENERAL_DIALOG_BODY": "Du er i ferd med å slå av mobildata i Smittestopp. Du vil ikkje bli varsla om potensiell smitte før du har Wi-Fi-tilkopling. Er du sikker på at du vil fortsette?",
   "SETTINGS_GENERAL_DIALOG_CANCEL": "Avbryt",
   "SETTINGS_GENERAL_DIALOG_OK": "OK",
   "SETTINGS_GENERAL_DIALOG_TITLE": "Åtvaring",
   "SETTINGS_GENERAL_TITLE": "Innstillingar ",
   "SETTINGS_GENERAL_EXPLANATION_ONE": "Her kan du redigere innstillingane dine",
   "SETTINGS_GENERAL_EXPLANATION_TWO": "Tilgang til mobildata",
-  "SETTINGS_GENERAL_MOBILE_DATA_HEADER": "Smittestopp må få tilgang til mobildata",
+  "SETTINGS_GENERAL_MOBILE_DATA_HEADER": "La Smittestopp bruke mobildata",
   "SETTINGS_GENERAL_MOBILE_DATA_DESC": "Når du slår av mobildata her, kan du styre databruken din, sidan det da berre blir henta informasjon om smitta du har vore i nærleiken av når du har vore tilkopla Wi-Fi. Appen er aktiv sjølv om du slår av mobildata.",
-  "SETTINGS_GENERAL_CHOOSE_LANGUAGE_HEADER": "Vel språk / Choose language ",
+  "SETTINGS_GENERAL_CHOOSE_LANGUAGE_HEADER": "Vel språk / Select language ",
   "SETTINGS_GENERAL_NB": "Bokmål",
   "SETTINGS_GENERAL_NN": "Nynorsk",
   "SETTINGS_GENERAL_EN": "English",
-  "SETTINGS_GENERAL_RESTART_REQUIRED_TEXT": "Start appen på nytt for å sjå språket du har valt / Restart the app to see your language setting",
+  "SETTINGS_GENERAL_RESTART_REQUIRED_TEXT": "Start appen på nytt for å sjå språket du har valt / Restart the app to apply your langauge selection.",
```

Comment:
Just a question. What with the feature of automatic restart after language change? Maybe customer should reconsider it?